pkg/monitortestlibrary/allowedalerts/basic_alert.go

```diff
@@ -72,7 +74,20 @@ func newAlert(bugzillaComponent, alertName string, jobType *platformidentificati
 	}
 }
 
-func newNamespacedAlert(alertName string, jobType *platformidentification2.JobType) *alertBuilder {
+// newAlertTestWithNamespace creates a single alert test within a specific namespace.
+func newAlertTestWithNamespace(bugzillaComponent, alertName, alertNamespace string, jobType *platformidentification2.JobType) *alertBuilder {
```

Comment:
The builder pattern we use to build the tests is neat.  Why we didn't continue the builder pattern for this and the per namespace one below? It'd feel more consistent to me.

i.e., 

` newAlertTest("bz-etcd", "etcdInsufficientMembers", jobType).inNamespace("kube-system").firing().toTests()...`

Reply:
I missed that, thanks!